app/clients/coin-market-cap/client.go

```diff
@@ -54,6 +54,8 @@ func NewClient(apiCfg config.CoinMarketCap) *Client {
 }
 
 func (c *Client) GetUsdPrices(idsByNetworkAndAddress map[uint64]map[string]string) (pricesByNetworkAndAddress map[uint64]map[string]decimal.Decimal, err error) {
```

Comment:
Remove the name declarations from here if you're explicitly initializing the map

Reply:
But this way I'll need to do `var err` for error when it is easier and cleaner if I have the name in the declaration. I think this way is okay, to be honest..